docs/UI/SetMediaClockTimer/index.md

```diff
@@ -33,11 +33,11 @@ The UI.SetMediaClock timer request indicates either an initial value for the med
     * Clear `startTime` to 00:00:00 in the case that the `startTime` parameter is not provided in the request, otherwise, `startTime` must be updated with a new value. It is up to HMI to determine the way the media clock timer is cleared: either to remove it from display or to set it to zero.   
 
```

Comment:
(MInor) The point after `Pause the timer that is counting up or down` don't appear as sub bullets. maybe add another indent or use `-` instead of `*`